docs/src/components/SdkVersions.js

```diff
@@ -0,0 +1,26 @@
+import { usePluginData } from '@docusaurus/useGlobalData';
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+import { maxSatisfying } from 'semver';
+
+export const Version = ({ sdk, spec = '*', options = {} }) => {
```

Comment:
Based on the below, do we actually need to export this function anymore?

Reply:
Fixed